clients/web/src/templates/body/plugins.jade

```diff
@@ -9,19 +9,25 @@ div.g-plugin-restart
   .g-plugin-clear
 .g-plugin-list-container
   each plugin in allPlugins
+    - var experimental = plugin.value.version ? plugin.value.version.indexOf('0.') === 0 : true
     .g-plugin-list-item
-      input.g-plugin-switch(type="checkbox", key="#{plugin.key}",
+      input.g-plugin-switch(type="checkbox", key=plugin.key,
                             disabled=!plugin.value.meetsDependencies,
-                            checked=(plugin.value.enabled ? "checked" : undefined))
+                            checked=(plugin.value.enabled ? "checked" : undefined),
+                            experimental=(experimental ? "true" : undefined),
+                            data-on-color=(experimental ? "warning" : "primary"),
+                            data-label-text=(experimental ? "&#xe95c;" : "&nbsp;"))
       .g-plugin-name
-        | #{plugin.value.name}
-        if (plugin.value.version)
-          span.g-plugin-version
-            | Version #{plugin.value.version}
-        if(plugin.value.configRoute)
-          a.g-plugin-config-link(g-route="#{plugin.value.configRoute}",
-                                 title="Configure Plugin")
+        span= plugin.value.name
+        if plugin.value.version
+          span.g-plugin-version Version #{plugin.value.version}
+        if experimental
+          span.g-experimental-notice(
+            title="WARNING: this plugin is an alpha version and is subject " +
+                  "to breaking changes; use at your own risk.") Experimental
+        if plugin.value.configRoute
+          a.g-plugin-config-link(g-route=plugin.value.configRoute, title="Configure Plugin")
             i.icon-cog
       .g-plugin-description #{plugin.value.description}
         if (!plugin.value.meetsDependencies)
-          .g-plugin-warning This plugin has unmet dependencies.
+          .g-plugin-warning.g-plugin-error-message This plugin has unmet dependencies.
```

Comment:
To some degree, we consider element class names to be part of our API surface, for the sake of selectors, not for style rules. Our web client plugins pattern of extension often involves wrapping a render method and then injecting elements into a page, and as such, these selectors become de facto hooks for such injection.

This isn't to say we'll *never* remove DOM elements or classes, just that we won't do it lightly.